project/SparkCassandraConnectorBuild.scala

```diff
@@ -59,14 +61,82 @@ object CassandraSparkBuild extends Build {
         "org.scala-lang" % "scala-compiler" % scalaVersion.value))
   ).disablePlugins(AssemblyPlugin, SparkPackagePlugin) configs IntegrationTest
 
-  lazy val connector = CrossScalaVersionsProject(
+  /**
+    * Do not included shaded dependencies so they will not be listed in the Pom created for this
+    * project.
```

Comment:
`sbt/sbt connectorDistribution:publish-local` produced pom containing
```
        <dependency>
            <groupId>com.google.guava</groupId>
            <artifactId>guava</artifactId>
            <version>16.0.1</version>
            <scope>provided</scope>
        </dependency>
```
is that ok?